### ISSUE TITLE
move to sklearn nightly builds?

### ISSUE THREAD

**Issue by amueller** [lesteve/scikit-learn-tests-pyodide#3]
Should we add this to sklearn CI? What are the hurdles?
@thomasjpfan said he's ok with having it fail on a nightly ;)

**Comment by rth** [comment]
There is a similar plan to add it to numpy CI in https://github.com/numpy/numpy/pull/21895 

So scikit-learn nightly could also make sense eventually. But to start, and make the CI pass it's indeed probably easier to have this standalone repo.

BTW, pydantic-core has also been running it in CI for some time now.

**Comment by lesteve** [comment]
I agree that a standalone repo makes it a lot easier to quickly iterate on this and see what is the state we are in before thinking of adding a CI build to the scikit-learn/scikit-learn repo.

When https://github.com/scikit-learn/scikit-learn/pull/24682 is merged we can get at least `pyodide build` to work on the scikit-learn `main` branch which is a start.

About hurdles, there are plenty of test failures right now that needs investigating. See https://github.com/lesteve/scikit-learn-tests-pyodide#manually-curated-list-of-issues for more details.
